lochness/mindlamp/__init__.py

```diff
@@ -215,6 +330,10 @@ def get_sensor_events_lamp(lamp: LAMP, subject_id: str) -> List[dict]:
     Returns:
         activity_dicts: activity records, list of dict.
     '''
-    sensor_event_dicts = lamp.SensorEvent.all_by_participant(subject_id)['data']
-
+    if not from_ts == None:
+        sensor_event_dicts = lamp.SensorEvent.all_by_participant(
+                        subject_id, _from=from_ts, to=to_ts)['data']
```

Comment:
Can you confirm that `LAMP.SensorEvent.all_by_participant` function has an argument `_from` but `to` without `_`?

Reply:
yes I confirm the API works with `_from` and `to`